src/NavBar/NavBar.tsx

```
@@ -0,0 +1,28 @@
+const paths: string[] = ["movie", "actor", "profile"];
+
+const NavBar: React.FC = () => {
+  return (
+    <div className="flex justify-center py-4">
+      <div className="container flex items-center justify-between">
+        <NavLink to="/">Home</NavLink>
+
+        <nav>
+          <ul className="flex gap-4">
+            {paths.map((path, index) => (
+              <li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)